test/sandbox-test.js

```diff
@@ -243,6 +243,28 @@ describe("sinonSandbox", function () {
         }
     });
 
+    describe(".restore", function () {
+        it("throws when passed arguments", function () {
+            this.sandbox = sinonSandbox.create();
+
+            var failed = false;
```

Comment:
Ah, we might need to update the dependencies for that package. Maybe it's not included in that version. I'll try to keep that info stored somewhere :-)